committeeoversightapp/templates/committeeoversightapp/compare_current_committees_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block content %}
+  <div class="row justify-content-center align-items-center">
+    <div class="col-12 col-md-4 col-lg-3 text-md-right">
+        <h2 class="mb-4 ml-md-3 red-orange-color">{{page.title}}</h2>
+        <small class="ml-md-3"><i>Last updated: {% now "jS F Y H:i" %}</i></small>
+    </div>
+
+    <div class="col-12 col-md-8 col-lg-9 col-xl-6 pt-2">
+      {{ page.body|safe }}
+    </div>
+  </div>
+
+  {% include "partials/compare_current_committees_table.html" with committees=house_committees title="House of Representatives"%}
+
+  {% include "partials/compare_current_committees_table.html" with committees=senate_committees title="Senate"%}
```

Comment:
Are these partial templates the only part of `compare_current_committees_page.html` and `compare_committees_over_congresses.html` that differ? If so, would it work to factor out the rest of the template logic into base template with a block like `{% block table_content %}` that you could populate with these partial templates?

Reply:
Great point! Yes.